src/map/ai/helpers/pathfind.cpp

```diff
@@ -615,10 +613,10 @@ bool CPathFind::AtPoint(const position_t& pos)
 
 bool CPathFind::InWater()
 {
-    if (isNavMeshEnabled())
+    /*if (isNavMeshEnabled())
```

Comment:
If commenting this out, leave a message saying why, or remove the dead code

Reply:
Forgotten code path, resolved by calling PNavigation->isInWater()